### ISSUE TITLE
loss = loss_object(pred, gt_label)

### ISSUE THREAD

**Issue by songsong-L** [xinliy/dlg_tensorflow#1]
crossentropy = tf.keras.losses.CategoricalCrossentropy()
交叉熵损失的参数为（y_true, y_pred）
但是代码中是这样写的：
loss = loss_object(pred, gt_label)
为什么代码里需要反着写才对呢，我也试了，正着写恢复不出来。

**Comment by xinliy** [comment]
我差不多也忘了，这个是我去年在hw实验室实习的时候自己弄的。。具体细节我也忘记了，反正如果能收敛并且恢复input就能用了吧，不行就中间再加断点看看。
